### ISSUE TITLE
Format: lambda with wrong indent

### ISSUE THREAD

**Issue by bpasero** [Microsoft/TypeScript#1925]
Have code like the following:

```ts
export class Foo {
	protected bar() {
		return new Something((c, e, p) => {

		}, function () {
			// goes wrong
		});
	}
}
```

Format it

You end up with:

```ts
export class Foo {

	protected bar() {
		return new Something((c, e, p) => {

		}, function () {
				// goes wrong
			});
	}
}
```

**Comment by SaschaNaz** [comment]
Fixed by #3001?

**Comment by vladima** [comment]
seems so, doesn't repro on the latest bits. Thanks for noticing this!
